onnx_neural_compressor/algorithms/weight_only/rtn.py

```diff
@@ -99,21 +103,46 @@ def rtn_quantize(
 
             weight = quant_utils.pad_tensor(weight, group_size, k_blocks)
 
-            satisfy_MatMulNBits_condition = (
-                version.Version(ort.__version__) > constants.ONNXRT1161_VERSION and num_bits == 4
-            )
+            satisfy_MatMulNBits_condition = ort_version > constants.ONNXRT1161_VERSION and num_bits == 4
```

Comment:
For quant a node logic (either MatMulBits or standard DQ), it looks quite similar for different algorithms and should be able to decoupled with algo by a good abstraction.  is it feasible to refactor?